addons/account/views/product_view.xml

```diff
@@ -58,7 +58,7 @@
                     <field name="fiscal_country_codes" invisible="1"/>
                 </xpath>
                 <page name="inventory" position="after">
-                    <page string="Accounting" name="invoicing" groups="account.group_account_readonly,account.group_account_invoice">
+                    <page string="Accounting" name="invoicing" groups="account.group_account_readonly">
                         <group name="properties" groups="account.group_account_readonly">
```

Comment:
```suggestion
                        <group name="properties">
```
We don't need to repeat the same `groups` attribute on child elements (several occurrences) :wink:

Reply:
Done for other child elements! just need `groups` attribute on this element because we are change parent element `groups` of parent element :slightly_smiling_face:

Reply:
Not sure I'm following you :sweat_smile: 
To me, the `groups` attribute of the `properties` `group` is identical to its parent `page`, no ? :man_shrugging: 
Is it not `account.group_account_readonly` in both instances ?
Or maybe you need to override it somewhere else ?

Reply:
Done!

Reply:
Hello @kcv-odoo 
I discussed this with people from the team, and I think this is not the way the issue should be addressed.
We don't really want to go down this path, because it will quickly become very complicated to maintain with all the inheritance.
The best way would be for the JS to simply not display pages where no field is visible.
I will check with the JS team if it can happen :wink:

Reply:
@epictete @kcv-odoo A task (3207900) has been started in the JS team to have the JS hide a tab if all fields are invisible for the user.

Reply:
Hey @chklop Great! I'll remove code related to that point from this PR :slightly_smiling_face: